src/OpenTelemetry.Instrumentation.AWSLambda/Implementation/AWSLambdaHttpUtils.cs

```diff
@@ -44,8 +44,9 @@ internal class AWSLambdaHttpUtils
         {
             case APIGatewayProxyRequest request:
                 httpScheme = AWSLambdaUtils.GetHeaderValues(request, HeaderXForwardedProto)?.LastOrDefault();
-                httpTarget = string.Concat(request.RequestContext?.Path ?? string.Empty, GetQueryString(request));
-                httpMethod = request.HttpMethod;
+                var path = request.Path ?? request.RequestContext?.Path ?? string.Empty;
+                httpTarget = string.Concat(path, GetQueryString(request));
+                httpMethod = request.HttpMethod ?? request.RequestContext?.HttpMethod;
```

Comment:
Should we also consider the case where the method is null/empty? I guess in that case it does not make sense to set any (other) http attributes

Reply:
I would consider it more general: should we set all required tags of one (or more) of required are missing ? But then it makes code more complex. I think we still should be more tolerant and set as more as we can.